docs/contributing/guidelines-for-phing-targets.md

```diff
@@ -62,6 +62,9 @@ For example, all paths to executables should be used via `path.*.executable` pro
 Sometimes, it can be even useful to include a `<arg line="${my-target.flags}"/>` in an `exec` task with an empty `my-target.flags` property provided.
 This will allow the end users to provide their own flags, modifying the command's behavior without the need for overwriting the whole target.
 
+Removing [DRY violations](/docs/contributing/code-quality-principles.md#dont-repeat-yourself) in the phing target definitions by extracting common tasks and sequences into their own (possibly hidden) targets also helps with extensibility.
+Instead if having to overwrite a few targets to change the definition, in such case users could overwrite just the extracted part.
```

Comment:
i am not sure about the structure of this sentence.

Reply:
I rewrote it in 646b30b02